frontend/src/MessageBoard/components/AddMessage.js

```diff
@@ -0,0 +1,201 @@
+import React from 'react';
+import styled from 'styled-components';
+import gql from 'graphql-tag';
+import { Mutation } from 'react-apollo';
+import { FilePond } from 'react-filepond';
+import '../../../node_modules/filepond/dist/filepond.min.css';
```

Comment:
Trying to import files in React from outside of the src/ folder will break your build. Please either copy the file into src/ somewhere, or integrate it into a styled component, being sure to license it properly in either case.